tests/security/stig/oscap_xccdf_eval.pm

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test "stig" hardening in the "scap-security-guide": detection mode
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93886, poo#104943
+
+use base 'stigtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # Get ds file and profile ID, ds file
```

Comment:
Suggest change 
# Get ds file and profile ID of ds file?

Reply:
Deleted ", ds file" (typo)